src/app/organizations/organization-form/organization-form.component.html

```diff
@@ -26,11 +26,13 @@
     <p-dropdown
       id="selectedBillingEntity"
       formControlName="billingEntity"
-      optionLabel="spec.name"
-      [options]="billingEntities"
+      optionLabel="label"
```

Comment:
I think `optionLabel="label"` is the default, so might be able to leave it out altoghether

Reply:
It actually _is_ required, otherwise the dropdown doesn't show the initial value set in the formbuilder